stable/prometheus-cloudwatch-exporter/README.md

```diff
@@ -0,0 +1,80 @@
+# Cloudwatch exporter
+
+* Installs [cloudwatch exporter](http://github.com/prometheus/cloudwatch_exporter)
+
+## TL;DR;
+
+```console
+$ helm install stable/prometheus-cloudwatch-exporter
+```
+
+## Introduction
+
+This chart bootstraps a [cloudwatch exporter](http://github.com/prometheus/cloudwatch_exporter) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- [kube2iam](../../stable/kube2iam) installed to used the **aws.role** config option otherwise configure **aws.aws_access_key_id** and **aws.aws_secret_access_key**
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ # edit aws.aws_access_key_id and aws.aws_access_key_id with the key/password of a AWS user with a policy to access  Cloudwatch
+$ helm install --name my-release stable/prometheus-cloudwatch-exporter
+$ # or add a role to aws with the [correct policy](https://github.com/prometheus/cloudwatch_exporter#credentials-and-permissions) to add to cloud watch
+$ helm install --name my-release stable/prometheus-cloudwatch-exporter --set awsRole=roll_name_here
+```
+
+The command deploys Cloudwatch exporter on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following tables lists the configurable parameters of the Cloudwatch Exporter chart and their default values.
+
+| Parameter                       | Description                                | Default                                                    |
```

Comment:
Please format the table properly. Either align columns or completely remove excess whitespace.